keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/tx/DeliveryConfigTransactionTests.kt

```diff
@@ -143,27 +143,6 @@ internal class DeliveryConfigTransactionTests : JUnit5Minutests {
       }
     }
 
-    context("an artifact attached to the delivery config fails to persist") {
```

Comment:
This is no longer an assumption - if an artifact fails to be persisted, we won't fail to persist the whole config.

Reply:
Is that bad? You'd have a bunch of resources and no artifact? I think the inverse is fine… having an artifact nothing references but this seems like it might cause problems.

Reply:
reverting this behavior after testing + in person conversations